data/human/hails.txt

```diff
@@ -1803,6 +1876,60 @@ phrase "friendly navy"
 		"in the Republic"
 	word
 		"!"
+		
+phrase "friendly navy"
```

Comment:
I feel like this whole thing might work better as just one or two static phrases. A lot of the combinations feel a bit awkward to me.

Reply:
I think the issue was mostly with the 2 removed entries and with the second part. I have changed them a bit, hope they are better now.